_sass/quiz-feedback.scss

```
@@ -0,0 +1,99 @@
+    background-color: $color-dark-incorrect-bg;
+    border-color: $color-dark-incorrect-border;
+    color: $color-dark-incorrect-text;
+  }
+
+  .quiz-feedback {
+    box-shadow: 0 2px 8px $color-dark-shadow;
+  }
+}
+
+// Animations
+@keyframes correctPulse {
```

Stylelint (17.4.0)

[error] Expected keyframe name "correctPulse" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)